qdrant-landing/content/documentation/frameworks/canopy.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Pinecone Canopy
+weight: 2500
+---
+
+# Pinecone Canopy
+
+[Canopy](https://github.com/pinecone-io/canopy) is an open-source framework and context engine to build chat assistants at scale.
+
+Qdrant is supported as a knowledge base within Canopy for context retrieval and augmented generation.
+
+## Usage
+
+Install the SDK with the Qdrant extra.
+
+```bash
+pip install canopy-sdk[qdrant]
+```
+
+### Creating a knowledge base
+
+```python
+from canopy.knowledge_base import QdrantKnowledgeBase
+
+kb = QdrantKnowledgeBase(collection_name="<YOUR_COLLECTION_NAME>")
+```
+
+<aside role="status">The constructor accepts additional <a href="https://github.com/qdrant/qdrant-client/blob/eda201a1dbf1bbc67415f8437a5619f6f83e8ac6/qdrant_client/qdrant_client.py#L36-L61">options</a> to customize your connection to Qdrant.</aside>
+
+To create a new Qdrant collection and connect it to the knowledge base, you can use the `create_canopy_collection` method:
```

Comment:
Not a "big deal," but you can be more direct, by using fewer "you ..."s (In grammar, it's known as the "imperative.")

```suggestion
To create a new Qdrant collection and connect it to the knowledge base, use the `create_canopy_collection` method:
```